scripts/aiven/cli-missing-docs.py

```diff
@@ -0,0 +1,72 @@
+
+import os
+from aiven.client import cli
+from docutils.core import publish_doctree
+
+
+# This function parses the CLI document folder and retrieves all the titles (matching the aiven command)
+def find_cli_docs():
+    list_of_commands = []
+    dirname = os.path.dirname(__file__).replace('scripts/aiven','') + "docs/tools/cli"
+    print(dirname)
+    folder_list = []
+    folder_list.append(dirname)
+    # Parsing all the folders under /docs/tools/cli
+    for folder in folder_list:
+        files = os.listdir(folder)
+        for fine_name in files:
+            # If we find subfolder, we add it to the list of folders
+            if os.path.isdir(folder + "/" + fine_name) and fine_name != '/':
+                folder_list.append(folder + "/" + fine_name)
+            # if it's a file read it, and parse the doctree to extract the titles
+            elif fine_name != '/':
+                with open(os.path.join("files", folder + "/" + fine_name), 'r') as f:
+                    text = f.read()
+                    titles = publish_doctree(text).traverse(condition=section_title)
+                    for t in titles:
+                        list_of_commands.append(t.astext())
+    # Remove from the titles the avn prefix to match it with the functions
+    list_of_commands = list(map(lambda x: x.replace('avn ', ''), list_of_commands))
+    
+    return sorted(list_of_commands)
+
+# This function parses the CLI and returns all the functions
+def find_cli_func():
```

Comment:
This is pretty close, but includes some things we don't want.

The things we *do* want are annotated with `@arg` in the aiven_client source code, so it feels like we should be able to use that.

Looking at the `help` method in `aiven/client/cli.py`, it is doing:
```
321:               arg_list = getattr(prop, argx.ARG_LIST_PROP, None)
```
to get back "the stuff stored by `@arg`", and then:
```
 322:               if arg_list is not None:
```
to ignore the method if it does not have such "stuff".

(we can ignore the `patterns` code just thereafter, as it's part of allowing one to do `avn help pg`, which I didn't know one could do)

(An alternative approach would be to generate the `avn help` output, by calling  `cli.AivenCLI().run(['help'])`, but it outputs to `stdout` (that could be captured) and also includes the summary text (which would be a pain to remove), so I ruled that out.)